tornado-runtime/src/main/java/uk/ac/manchester/tornado/runtime/common/TornadoOptions.java

```diff
@@ -459,4 +468,8 @@ private static boolean isFPGAEmulation() {
         String contextEmulatorXilinxFPGA = System.getenv("XCL_EMULATION_MODE");
         return (contextEmulatorIntelFPGA != null && (contextEmulatorIntelFPGA.equals("1"))) || (contextEmulatorXilinxFPGA != null && (contextEmulatorXilinxFPGA.equals("sw_emu")));
     }
+
+    public static boolean cleanUpAtomicsSpace() {
```

Comment:
can we document also this?

Reply:
Yes, good point. I just pushed the javadoc.